src/utils/internal/parseMultipartData.test.ts

```diff
@@ -1,6 +1,24 @@
 import { parseMultipartData } from './parseMultipartData'
 
 test('parses a given valid multipart string', async () => {
+  expect.assertions(3)
+  await testMultipartDataWithContentType(
+    'multipart/form-data; boundary=WebKitFormBoundaryvZ1cVXWyK0ilQdab',
+  )
+})
+
+test('parses a given valid multipart string give non-pretty content-type', async () => {
```

Comment:
Looks like we need one more scenario when there are multiple spaces between the content type and its additional information: `multipart/form-data;     boundary=xxx`.

Reply:
done